src/redux/configureStore.js

```diff
@@ -7,6 +8,6 @@ const store = configureStore({
     books,
     categories,
   },
-});
+}, applyMiddleware(thunk));
```

Comment:
- [ ] Please use [createAsyncThunk](https://redux-toolkit.js.org/api/createAsyncThunk) directly in your redux slice as the project requirements mentioned because the new redux toolkit already has thunk configured in it.

![image](https://user-images.githubusercontent.com/68796232/179252843-3cbfaf27-312d-4113-baf9-1bd8f2d71ad6.png)

Reply:
DOne